hackIDE/static/hackIDE/js/custom.js

```diff
@@ -269,7 +290,13 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						$(".error-message").html(response.compile_status);
+
```

Comment:
@jarroyo3 would you be willing to separate these two changes in two different PRs? That would be helpful in easy reviewing and managing the code commits history :)